### PR TITLE
Mv cache warming tool  [JIRA: RIAK-2351]

### PR DESCRIPTION
**Pull request by matthewvon**
Internal / consulting tool for creating an initial cache warming file.  This is NOT industrial grade and requires hand adjustments to shell script depending upon the user environment.

### CODE REVIEW

**Comment by matthewvon** [comment]
Original comments from Paul Place's code review:

I didn't apply the same standards as I would to production code, since my understanding is that the purpose of this tool is to ensure the new code in mv-cache-object-warming has a COW file the first time it runs after upgrading to this branch.

All looks fine except for the following in tools/cache_warm.cc
    line 129: NAME_MAX does not include the terminating null character, so should make it file[NAME_MAX+1]
    line 264: need to update the comman_help() method from sst_rewrite to cache_warm

--Paul

File: tools/cache_warm.cc [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        }   // if
+
+        // file of .sst pathnames
+        else
+        {
+            FILE * list;
+            bool ignore_file;
+            std::string cur_path, record;
+            size_t line_count;
+            char buffer[PATH_MAX], vnode[PATH_MAX], file[NAME_MAX], * chr_ptr;
```
Comment (matthewvon) [inline tools/cache_warm.cc]: Paul Place writes: NAME_MAX does not include the terminating null character, so should make it file[NAME_MAX+1]

File: tools/cache_warm.cc [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    if (1==argc)
+        command_help();
+
+    return( error_seen ? 1 : 0 );
+
+}   // main
+
+
+void
+command_help()
```
Comment (matthewvon) [inline tools/cache_warm.cc:263]: Paul Place writes:  need to update the comman_help() method from sst_rewrite to cache_warm

**Comment by bashopatricia** [comment]
create jira issue

**Comment by Basho-JIRA** [comment]
1/21 per MVM, not ready for this release. 2.2 or 2.0.7 

**_[posted via JIRA by Patricia Brewer]**_

**Pull request closed without merging**